src/engine/client/ClientApplication.cpp

```diff
@@ -93,9 +96,9 @@ class ClientApplication : public Application {
         }
 
         void Shutdown(bool error, Str::StringRef message) override {
-            #if defined(_WIN32) || defined(BUILD_GRAPHICAL_CLIENT)
```

Comment:
Was the windows32 define due to windows spawning a terminal automatically?

Reply:
I think we would use something from the platform abstractions SDL provides? I've never tried running daemonded on windows, but I would be surprised if SDL prints something in a terminal.

If you think it can I can find a somewhat different hack

Reply:
On Windows daemonded/daemon-tty don't use a real console; PDCurses creates a regular window and draws the curses interface in it. The messagebox is needed because if the program just exits there is no chance to read the error message.

Reply:
Ok I've added the call back.